ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.c

```diff
@@ -57,6 +67,24 @@ tflite_close (const GstTensorFilterProperties * prop, void **private_data)
   *private_data = NULL;
 }
 
+/**
+ * @brief return nnapi_hw type
+ */
+static nnapi_hw
+get_nnapi_hw_type (const gchar * str)
+{
+  gint index = 0;
+
+  while (nnapi_hw_string[index]) {
+    if (!g_ascii_strcasecmp (nnapi_hw_string[index], str)) {
+      return index;
+    }
+    index++;
+  }
```

Comment:
Add a null-terminator to nnapi_hw_string and reuse find_key_strv from tensor_common.h

Reply:
done.